cmd/updater/subcommands/bootstrap/command_test.go

```diff
@@ -0,0 +1,21 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package bootstrap
+
+import (
+	"testing"
+
+	"github.com/DataDog/datadog-agent/cmd/updater/command"
+	"github.com/DataDog/datadog-agent/pkg/util/fxutil"
+)
+
+func TestCommand(t *testing.T) {
+	fxutil.TestOneShotSubcommand(t,
+		Commands(&command.GlobalParams{}),
+		[]string{"bootstrap"},
+		bootstrap,
+		func() {})
+}
```

Comment:
Could we add a test case that validate that if the if the remote config service is not running we return the correct error?

Reply:
Tested directly in the component at https://github.com/DataDog/datadog-agent/pull/22871/commits/0227ec616e78f2571c5ab96e805e14c8a7837c69